source/ui-thymeleaf/build.gradle.kts

```diff
@@ -13,6 +13,9 @@ dependencies {
 
     implementation("org.springframework.boot:spring-boot-starter-thymeleaf")
     implementation("org.springframework.boot:spring-boot-starter-web")
+    implementation(project(mapOf("path" to ":source:firm:api:in")))
+    implementation(project(mapOf("path" to ":source:firm:api:in")))
```

Comment:
why 3 times?

Reply:
why project(mapOf("path" to ":source:firm:api:in") but not  implementation(project(":source:firm:api:in")) ??